index.html

```diff
@@ -1,269 +1,392 @@
+                src="./assets/images/portfolio-1.png"
+                alt="Multi-Post Stories Gain+Glory"
+                class="thumbnail"
+              />
+              <div class="portfolio-text">
+                <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
                 <ul class="categories">
-                    <li class="category">Codekit</li>
-                    <li class="category">GitHub</li>
-                    <li class="category">Codepen</li>
-                    <li class="category">Gitlab</li>
-                    <li class="category">Terminal</li>
+                  <li class="category">Ruby on rails</li>
+                  <li class="category">css</li>
+                  <li class="category">Javascript</li>
+                  <li class="category">html</li>
                 </ul>
-                
+                <a href="#" class="button btn-primary">See Project</a>
+              </div>
             </div>
+          </div>
         </div>
-    </div>
-</div>
-        </section>
-        <footer id="contact">
-            <div class="section-content container contact-me">
-            <h2 class="section-heading neutrals txt-center">
-                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. 
-            </h2>
-            <div class="mobile-form">
-                <form class="my-contact-form"
-                action="https://formspree.io/f/xdojwqqv"
-                method="POST"
-                >
-                    <input type="text" name="name" maxlength="30" placeholder="Full name" required>
-                    <input type="email" name="email" placeholder="Email Adress" required>
-                    <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
-                    <button class="button btn-primary border-none" type="submit">Get In touch</button>
-                </form>
-            </div>
-            <div class="desktop-form">
-                <form class="my-contact-form"
-                action="https://formspree.io/f/xdojwqqv"
-                method="POST"
-                >
-                    <fieldset>
-                    <input type="text" name="first-name" maxlength="30" placeholder="First name" required>
-                    <input type="text" name="last-name" maxlength="30" placeholder="Last name" required>
-                    </fieldset>
-                    <input type="email" name="email" placeholder="Email Adress" required>
-                    <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
-                    <button class="button btn-primary border-none" type="submit">Get In touch</button>
-                </form>
+      </section>
+      <section class="about-me" id="about">
+        <div class="section-content container">
+          <div class="about-text">
+            <p class="section-heading txt-center">About Me</p>
```

Comment:
- [ ] Kindly make this title `h2` to make these work for everyone, the headings need to be marked up. That way people can navigate to the headings — including people who cannot use a mouse and use only the keyboard, and people who use a screen reader.

Reply:
Ok, my partner mistakenly edited the wrong tag. I'm on it.